pyphare/pyphare/pharesee/hierarchy/fromvtkhdf5.py

```
@@ -0,0 +1,488 @@
+
+    if load_one_time(time, hier):
+        time = listify(time)
+        assert len(time) == 1
+        return add_time_from_h5(hier, h5_filename, time[0], **kwargs)
+
+    if load_all_times(time, hier):
+        for t in VtkFile(h5_filename).times:
+            add_time_from_h5(hier, h5_filename, t, **kwargs)
+        return hier
+
+    assert False
```

Ruff (0.14.11)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)